azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,327 @@
+
+```Python
+url = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/Broadway_and_Times_Square_by_night.jpg/450px-Broadway_and_Times_Square_by_night.jpg"
+
+image_analysis = client.analyze_image(url,visual_features=[VisualFeatureTypes.tags])
+
+for tag in image_analysis.tags:
+    print(tag)
+```
+
+### Get subject domain list
+
+Review the subject domains used to analyze your image with [`list_models`][ref_computervisionclient_list_models]. These domain names are used when [analyzing an image by domain](#analyze-an-image-by-domain). An example of a domain is `landmarks`.
+
+```Python
+models = client.list_models()
+
+for x in models.models_property:
+    print(x)
+```
+
+### Analyze an image by domain
+
+You can analyze an image by subject domain with [`analyze_image_by_domain`][ref_computervisionclient_analyze_image_by_domain]. Get the [list of supported subject domains](#get-subject-domain-list) in order to use the correct domain name.  
+
+```Python
+domain = "landmarks"
+url = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/12/Broadway_and_Times_Square_by_night.jpg/450px-Broadway_and_Times_Square_by_night.jpg"
+language = "en"
+
+analysis = client.analyze_image_by_domain(domain, url, language)
+
+for landmark in analysis.result["landmarks"]:
+    print(landmark["name"])
+    print(landmark["confidence"])
+```
+
+### Get text description of an image
+
+You can get a language-based text description of an image with [`describe_image`][ref_computervisionclient_describe_image]. Request several descriptions with the `max_description` property if you are doing text analysis for keywords associated with the image. Examples of a text description for the following image include `a train crossing a bridge over a body of water`, `a large bridge over a body of water`, and `a train crossing a bridge over a large body of water`.
+
+```Python
+domain = "landmarks"
+url = "http://www.public-domain-photos.com/free-stock-photos-4/travel/san-francisco/golden-gate-bridge-in-san-francisco.jpg"
+language = "en"
+max_descriptions = 3
+
+analysis = client.describe_image(url, max_descriptions, language)
+
+for caption in analysis.captions:
+    print(caption.text)
+    print(caption.confidence)
+```
+
+### Get text from image
+
+You can get any handwritten or printed text from an image. This requires two calls to the SDK: **[recognize_text][ref_computervisionclient_recognize_text]** and **[get_text_operation_result][ref_computervisionclient_get_text_operation_result]**. The call to recognize_text is asynchronous. In the results of the get_text_operation_result call, you need to check if the first call completed with [`TextOperationStatusCodes`][ref_computervision_model_textoperationstatuscodes] before extracting the text data. The results include the text as well as the bounding box coordinates for the text. 
```

Comment:
I dig the backtick format you've used elsewhere for the deep links to ref content; curious about your reasoning for going with bold on these two.

Reply:
I wanted to call out that there were 2 with the bold but I'll change it.